docs/contributing.md

```diff
@@ -1,58 +1,61 @@
-# Developer Contributor Guide
+# Contributor Guide
```

Comment:
There is another document somewhere for general non-dev contributing too. And this is very developer focussed, so this should probably be the dev contributor guide